assets/js/components/Dashboard/AlertBanner.tsx

```diff
@@ -54,7 +82,8 @@ const AlertBanner: ComponentType = () => {
               alert was just closed.`}
             </span>{" "}
             Closing an alert may cause others to appear on more screens. It
-            could take up to 40 seconds for any impacted screens to update.
+            could take up to {timeLeft} seconds for any impacted screens to
```

Comment:
Super nit-picky, but should there be logic here to drop the "s" when `timeLeft === 1`?